.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,200 @@
+name: Helm Multi Release
+# Reusable workflow for packaging and publishing Helm charts having multiple subcharts on GitHub pages
+
+on:
+  workflow_call:
+    inputs:
+      charts-dir:
+        description: "The directory containing the Helm chart and `.bumpversion.cfg` file."
+        required: true
+        type: string
+      root-branch:
+        description: "The main branch."
+        default: "main"
+        required: false
+        type: string
+      subdirs:
+        description: "List of subdir to consider" # "['subdir1', 'subdir2', 'subdir3']"
+        required: true
+        type: string
+      artifact-dir:
+        description: "Directory inside `charts-dir` for preparation of the GitHub pages artifact."
+        required: false
+        default: "artifacts"
+        type: string
+      gh-pages-branch:
+        description: "Name of branch containing the artifacts"
+        default: "gh-pages"
+        type: string
+      project-root-dir:
```

Comment:
Why is it called `root-dir` when you want the user to input the `charts-dir`. I think `charts-dir` would more fitting